library/plugins/influxdb/influxdb2-publisher/src/main/kotlin/io/github/cdsap/talaiot/publisher/influxdb2/InfluxDb2PublisherConfiguration.kt

```diff
@@ -0,0 +1,67 @@
+package io.github.cdsap.talaiot.publisher.influxdb2
+
+import io.github.cdsap.talaiot.configuration.PublisherConfiguration
+import io.github.cdsap.talaiot.metrics.BuildMetrics
+import io.github.cdsap.talaiot.metrics.TaskMetrics
+
+/**
+ * Configuration for the [InfluxDb2Publisher]. It belongs to the Publisher configurations
+ *
+ * influxDbPublisher {
+ *    token = "tracking"
```

Comment:
nit: can we replace the tracking label for something more representative like `System.getEnv($TOKEN)`